docs/Operators.md

```diff
@@ -17030,6 +17031,187 @@ expect(node, inputs=[x], outputs=[y],
 </details>
 
 
+### <a name="SoftmaxCrossEntropy"></a><a name="softmaxcrossentropy">**SoftmaxCrossEntropy**</a>
```

Comment:
```suggestion
### <a name="SoftmaxCrossEntropyLoss"></a><a name="softmaxcrossentropy">**SoftmaxCrossEntropy**</a>
```
SoftmaxCrossEntropy ---> SoftmaxCrossEntropyLoss

The first function also outputs normalization coefficient but the second term doesn't.